src/Executor/ScopedContext.php

```diff
@@ -0,0 +1,12 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Executor;
+
+/**
+ * This interface can be used to indicate that your context should be scoped.
```

Comment:
How does it indicate that? (I know, it is by implementing it in the value that is passed as the context, but that is not obvious).

Reply:
Fixed